joomla/joomla-cms
# [4.0] Normalise display of searchtools specific fields for menu items and multilingual associations managers

Redoing https://github.com/joomla/joomla-cms/pull/19076 but this time just to align the separate fields to the left of the Managers (LTR) and to the right (RTL) as it is done for other managers who have only one separate field (Template styles, Installed languages)

### Summary of Changes
Correcting Associations and Menu items searchtools Layouts
Correcting searchtools css to enable custom class when 2 specific fields.


### Testing Instructions
Install J as multilingual (You may have to do all manually since some stuff is broken with workflow.)
Create menu items and categories tagged to a language. ATM one can't create articles.
Display the Menu items Manager.
Display the Multilingual associations Manager

**MAKE SURE to delete** the `com_associations/layouts/joomla/searchtools/default/bar.php`

**PatchTester : NO. Needs npm install.**


### Before patch
<img width="1506" alt="screen shot 2018-08-13 at 08 43 35" src="https://user-images.githubusercontent.com/869724/44016380-06d17cac-9ed5-11e8-9a5e-78346a5f1c85.png">

![screen shot 2018-08-13 at 08 44 51](https://user-images.githubusercontent.com/869724/44016431-3b844c04-9ed5-11e8-89de-3a9b65acb03c.png)


### After patch
<img width="1465" alt="screen shot 2018-08-13 at 08 36 48" src="https://user-images.githubusercontent.com/869724/44016154-22b82f7a-9ed4-11e8-872a-c42d49e07479.png">

<img width="1463" alt="screen shot 2018-08-13 at 08 36 19" src="https://user-images.githubusercontent.com/869724/44016164-28f7f5e6-9ed4-11e8-8843-495ffb692387.png">

RTL
<img width="1447" alt="screen shot 2018-08-13 at 08 39 07" src="https://user-images.githubusercontent.com/869724/44016197-66a064d2-9ed4-11e8-990f-d5382103cede.png">

<img width="1506" alt="screen shot 2018-08-13 at 08 40 07" src="https://user-images.githubusercontent.com/869724/44016229-8411a63e-9ed4-11e8-9c41-0591dc47a5b6.png">


### Documentation Changes Required
